### PR TITLE
Add handwired Planck Mit and my personal layout

### PR DESCRIPTION
**Pull request by Wholteza**
I have an old handwired planck of the mit version that i bought in pieces from Jack Humbert back in 2015, I don't even think the rev1 pcb existed by then. This PR adds support for handwired plancks with a default + my personal layout since i could not find it in the repo already.

I am aware that the commits aren't super descriptive, so feel free to squash merge if needed.

This is my first pr to this project, i'm trying to follow all the documentation and guidelines as good as i can. I'm happy to get any constructive feedback.

I'm available in the QMK Discord server as well, feel free to @ me.

## Description

- Add support for planck under `handwired` with the mit layout.
- Add a default layout based upon the rev1 layout but removing functionality that wasn't there in the handwired version such as back-lighting.
- Add my own layout.
- Pinouts are defined in layout dir `config.h` since every handwireing can be different.

## Types of Changes

- [x] Keyboard addition
- [x] Keymap/layout addition

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
   - Since i am not the designer of the keyboard i added olkb and the link that the qmk documentation has to handwireing your own planck as origin and url in the info.json and keyboard readme. I just added myself as maintainer, is that ok? 
- [ ] My change requires a change to the documentation.
   - I have not seen any need to update documentation after reading the contribution guidelines.
- [ ] I have updated the documentation accordingly.
   - N/A 
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
   - I did not understand if it was ok to have the keyboard support and layout in the same PR, it it? 
- [ ] I have added tests to cover my changes.
   - As i understand the documentation, new kb and layouts don't have any unit tests?
   - My additions are purely declarative so i guess testing would be of the functionality using those additions?
   - In that case, N/A
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).
   - By making my keyboard with both my own and default layout.
   - I found that making all kb in the repo generates a lot of errors probably due to unmaintained kbs? Should not be originating from this pr at least.

### CODE REVIEW

**Comment by drashna** [comment]
Please review the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and verify that you have followed the checklist for the type of PR this is.

**Comment by Wholteza** [comment]
> Please review the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and verify that you have followed the checklist for the type of PR this is.

Yes i'm on my way. I didn't realize the pr would be visible for others during the draft. Will publish once ready. :+1:

File: keyboards/handwired/planck/config.h [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() (keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)))
+
+/* prevent stuck modifiers */
+#define PREVENT_STUCK_MODIFIERS
```
Comment (drashna) [inline keyboards/handwired/planck/config.h:53]:
```suggestion
```

File: keyboards/handwired/planck/config.h [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x6060
+#define DEVICE_VER 0x0001
+#define MANUFACTURER Handwired
+#define PRODUCT Planck
```
Comment (drashna) [inline keyboards/handwired/planck/config.h:26]: This name should be changed.

eg, `ortho_4x12`

File: keyboards/handwired/planck/config.h [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() (keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)))
+
+/* prevent stuck modifiers */
+#define PREVENT_STUCK_MODIFIERS
+
+#ifdef RGB_DI_PIN
+#    define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/handwired/planck/config.h:56]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/handwired/planck/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "planck.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    // QWERTY
+	KEYMAP(
```
Comment (drashna) [inline keyboards/handwired/planck/keymaps/default/keymap.c:20]:
```suggestion
	LAYOUT(
```

File: keyboards/handwired/planck/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "planck.h"
```
Comment (drashna) [inline keyboards/handwired/planck/keymaps/default/keymap.c:16]:
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/handwired/planck/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+    // QWERTY
+	KEYMAP(
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,
+        KC_ESC,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_ENT ,
+        MO(3),   KC_LCTL, KC_LALT, KC_LGUI, MO(1),       KC_SPC,       MO(2),   KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT
+    ),
+
+    // LOWER
+	KEYMAP(
```
Comment (drashna) [inline keyboards/handwired/planck/keymaps/default/keymap.c:28]:
```suggestion
	LAYOUT(
```

File: keyboards/handwired/planck/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+        KC_TILD, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR,    KC_ASTR,    KC_LPRN, KC_RPRN, KC_BSPC,
+        KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_UNDS,    KC_PLUS,    KC_LCBR, KC_RCBR, KC_PIPE,
+        KC_TRNS, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  S(KC_NUHS), S(KC_NUBS), KC_HOME, KC_END,  KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS,      KC_TRNS,    KC_MNXT,    KC_VOLD, KC_VOLU, KC_MPLY
+    ),
+
+    // RAISE
+	KEYMAP(
```
Comment (drashna) [inline keyboards/handwired/planck/keymaps/default/keymap.c:36]:
```suggestion
	LAYOUT(
```

File: keyboards/handwired/planck/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC,
+        KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_TRNS, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_NUHS, KC_NUBS, KC_PGUP, KC_PGDN, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS,      KC_TRNS, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY
+    ),
+    // EXTRAS
+	KEYMAP(
```
Comment (drashna) [inline keyboards/handwired/planck/keymaps/default/keymap.c:43]:
```suggestion
	LAYOUT(
```

File: keyboards/handwired/planck/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+
+	}
+
+	if (usb_led & (1 << USB_LED_KANA)) {
+
+	} else {
+
+	}
+
+}
```
Comment (drashna) [inline keyboards/handwired/planck/keymaps/default/keymap.c:94]:
```suggestion
```

File: keyboards/handwired/planck/keymaps/wholteza/keymap.c [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "planck.h"
+#include "keymap_swedish.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    // COLEMAK
+	KEYMAP(
```
Comment (drashna) [inline keyboards/handwired/planck/keymaps/wholteza/keymap.c:21]:
```suggestion
	LAYOUT(
```

**Pull request closed without merging**